conf.py

```diff
@@ -110,8 +110,12 @@
     'title': 'Page Not Found',
     'body': '''
 <h1>Page Not Found</h1>
-<p>Sorry, we couldn't find that page.</p>
-<p>Try using the search box or go to the homepage.</p>
+<p>Sorry, we couldn't find that page. Error code 404. </p>
+<p>You can try using the search box above or some of the handy links below:</p>
+<ol>
+  <li><a href="https://docs.jina.ai/chapters/CONTRIBUTING.html#">Contributing to Jina</a></li>
+  <li><a href="https://docs.jina.ai/chapters/CONTRIBUTING.html#join-us-on-slack">Join us on Slack</a><</li>
+</ol>
 ''',
 }
 notfound_no_urls_prefix = True
```

Comment:
Hi @FionnD your github creds don't seem to be configured, you can do `git commit --amend --author="Your name <email-id>" --no-edit` and push again so the commit show up with your authorship

Reply:
I'll setup CLI stuff for this repo.

Reply:
good catch @rutujasurve94 ! Done